cmd/samples/http/sender/main.go

```diff
@@ -44,13 +44,16 @@ func _main(args []string, env envConfig) int {
 		return 1
 	}
 
+	ctx := cloudevents.ContextWithEncoding(context.Background(), cloudevents.Structured)
+
 	seq := 0
 	for _, contentType := range []string{"application/json", "application/xml"} {
 		for _, encoding := range []cloudevents.HTTPEncoding{cloudevents.HTTPBinaryV01, cloudevents.HTTPStructuredV01, cloudevents.HTTPBinaryV02, cloudevents.HTTPStructuredV02} {
```

Comment:
I think we should add `HTTPBinaryV03`, `HTTPStructuredV03`, `HTTPBatchedV03` in the array